Updated ComboBox Styling topic. (#972)
telerik/maui-docs: controls/combobox/styling.md

The [Telerik UI for .NET MAUI ComboBox]({%slug combobox-overview%}) provides the following Style properties for customizing its look:

* `PlaceholderColor`(`Color`)&mdash;Defines the color for the placeholder text.

> * `TextColor`(`Color`)&mdash;Defines the color of the text when the control is editable and the color of the selected item when the control is not editable and the selection mode is single.

Reviewer: Can you split and simplify this sentence? For example:

* `TextColor`(`Color`)&mdash;Defines the following colors:
  * The color of the text when the control is editable.
  * The color of the selected item when the control is not editable and the selection mode is set to single.

* `HighlightTextColor`(`Color`)&mdash;Specifies the color of the text that will be highlighted when searching (`IsEditable` must be `True`).
* `BackgroundColor`(`Color`)&mdash;Defines the background color of the control.
* `BorderColor`(`Color`)&mdash;Defines the color of the border.
* `BorderThickness`(`Thickness`)&mdash;Defines the thickness of the border.
* `ClearButtonStyle` (of type `Style` with target type `Telerik.Maui.Controls.RadTemplatedButton`)&mdash;Defines the style for the clear button.
* `TextInputStyle`(of type `Style` with target type `Telerik.Maui.Controls.RadTextInput`)&mdash;Defines the style of the inner `RadTextInput` control used when the ComboBox is editable (`IsEditable` must be `True`).

> * `Font Options`(`FontAttributes`, `FontFamily`, `FontSize`)&mdash;Define the font options to the text of the ComboBox. It's applied to the Placeholder, Selected Text (for single selection), and when the control is in Editable Mode.

Reviewer: ```suggestion
* `Font Options`(`FontAttributes`, `FontFamily`, `FontSize`)&mdash;Defines the font options for the text of the ComboBox. It's applied to the Placeholder and Selected Text (for single selection) and when the control is in Editable Mode.
```

In addition, you can change the visual appearance of the ComboBox by defining the following visual states through the .NET MAUI Visual State Manager:

* **Normal**&mdash;Applied when the ComboBox is in its normal state.
* **Focused**&mdash;Applied when the ComboBox receives focus (when it's editable).
* (Windows Only) **MouseOver**&mdash;Applied when the mouse cursor is hovering over the ComboBox.
* **Disabled**&mdash;Applied when the ComboBox's `IsEnabled` is `False`.

### Example for ComboBox Styling

The example below demonstrates some of the styling capabilities of the ComboBox, such as custom `ClearButtonStyle`, `TextInputStyle`, `TextColor`, `PlaceholderColor`, and other, as well how you can switch its appearance through the .NET MAUI Visual State Manager.

> The example uses the same `ViewModel` and `City` classes as from the [Getting Started]({%slug combobox-getting-started%}) topic.

Reviewer: ```suggestion
The example uses the same `ViewModel` and `City` classes as the [Getting Started]({%slug combobox-getting-started%}) topic.
```

## DropDown Styling

The following properties style the ComboBox DropDown:

> * `DropDownBorderColor`(`Color`): Defines the color of the border around the drop down part of the control.

Reviewer: * In the list items, use an em dash as a separator instead of the colon.
* Use either "DropDown" instead of "drop down".

> * `DropDownBorderThickness`(`Thickness`): Defines the thickness of the border that is around of the drop down part of the control.

Reviewer: ```suggestion
* `DropDownBorderThickness`(`Thickness`)&mdash;Defines the thickness of the border that surrounds the DropDown part of the control.
```

> * `DropDownBorderCornerRadius`(`Thickness`): Defines the corner radius of the border that is around the drop down part of the control

Reviewer: ...that surrounds the...
